samples/MusicStore.Standalone/Program.cs

```diff
@@ -14,10 +16,13 @@ public static void Main(string[] args)
                 .AddEnvironmentVariables(prefix: "ASPNETCORE_")
                 .Build();
 
+            var currentExecutingAssemblyFileInfo = new FileInfo(typeof(Program).GetTypeInfo().Assembly.Location);
+
             var builder = new WebHostBuilder()
                 .UseConfiguration(config)
                 .UseIISIntegration()
-                .UseStartup("MusicStore.Standalone");
+                .UseStartup("MusicStore.Standalone")
+                .UseContentRoot(currentExecutingAssemblyFileInfo.Directory.FullName);
```

Comment:
To prevent duplicating logic from musicstore project, we copy the content files like Views, config files to the output folder of the standalone project and because of this we would have to change the location of the content root here.